docs/09-Configuration reference/02-Custom Resource Definitions.md

```
@@ -2554,6 +2555,307 @@ Once the reconciler has found the secret, it will copy it inside the stack names
+
+
+
+
+
+
+| Field | Description | Default | Validation |
+| --- | --- | --- | --- |
+| `name` _string_ |  |  |  |
+| `replicas` _string_ | Replicas: "auto" or integer as string |  |  |
+| `stateful` _boolean_ |  |  |  |
+| `containers` _[ContainerSpec](#containerspec) array_ |  |  |  |
```

markdownlint-cli2 (0.18.1)

Link fragments should be valid

(MD051, link-fragments)